frontend/packages/dev-console/src/components/import/image-search/ImageStream.tsx

```diff
@@ -80,7 +80,7 @@ const ImageStream: React.FC = () => {
   const helperTextInvalid = validated === ValidatedOptions.error && (
```

Comment:
We need to check the presence of isi.status.message as well.

![image](https://user-images.githubusercontent.com/9964343/75518904-05293600-5a28-11ea-8c28-040ce750dc5d.png)

Reply:
Yeah, missed it probably because there was no check for `isi.status` as well

Reply:
Exclamation Symbol is shown even if there is no message to show besides it.

Reply:
Yes, this happens in master as well.

Reply:
Adding the check